app/views/vocab_sheets/_vocab_sheet_item.html.haml

```diff
@@ -4,6 +4,10 @@
       = hidden_field_tag "item[#{vocab_sheet_item.id}][id]", vocab_sheet_item.id, class: 'item_id'
       = link_to sign_url(vocab_sheet_item.sign_id), { class: "drawing vocab_sheet_drawing drawing vocab_sheet_drawing__size--#{size}" } do
         = image_tag "/images/signs/400-400/#{vocab_sheet_item.drawing}"
+        = link_to "/images/signs/400-400/#{convert_to_high_resolution(vocab_sheet_item.drawing)}" do
```

Comment:
this repeats html from show.html.haml. How about making something re-usable that renders the images?